### DISCUSSION TITLE
Paperless won't finish update stuck at "Updating Ghostscript ..."

### DISCUSSION THREAD

**Discussion by yuriw** [community-scripts/ProxmoxVE#135] [General]
After starting the update I see Paperless won't finish update stuck at "Updating Ghostscript ..."

Any clues on how to work around this appreciated.
TIA

**Comment by Mellowlynx** [comment]
I'm testing the scripts commands now.
But it needs way longer then normal.

**Reply by Mellowlynx** [reply]
After running the ghostscript command by hand, the update could finish: 
![image](https://github.com/user-attachments/assets/d9bef0d6-5879-4e28-8ca2-6df965590529)
![image](https://github.com/user-attachments/assets/9b291dd1-03e5-4516-924a-59e326b23420)

**Reply by Mellowlynx** [reply]
@yuriw Can you run: `rm -rf /tmp/ghostscript*`? (removes the downloaded files from before)
And try to update again?
Just let it do its thing and give it 20 or so minutes.
(You can also try to give the LXC more specs)

**Reply by yuriw** [reply]
What commands did you run!

**Reply by yuriw** [reply]
> Can you run: `rm -rf /tmp/ghostscript*`
> 
> And try to update again?
> 
> Just let it do its thing and give it 20 to 30 minutes.
> 
> (You can also try to give the LXC more specs)

Will do soon

**Reply by yuriw** [reply]
@Mellowlynx it actually worked after ~10 min!

Odd ...  It used to do it much faster.

Thx for your help and testing.

**Reply by Mellowlynx** [reply]
It has to make the install, and that can take some time.
Great to hear that it works 😁

**Comment by Mellowlynx** [comment]
Made a pull request to make it more clear that it needs some time.

**Comment by MickLesk** [comment]
The Update runs 20-25min. Ghostscript is big and need time with make install. Not workaround yet, because debian repo has only 10.0, we need 10.4
